creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -175,12 +119,71 @@ const healthCheckVerbose = async ({ libs, snapbackSM } = {}, logger, sequelize,
     snapbackModuloBase,
     snapbackJobInterval,
     transcodeActive,
-    transcodeWaiting
+    transcodeWaiting,
+    fileProcessingActive,
+    fileProcessingWaiting
+  }
+
+  // If optional `randomBytesToSign` query param provided, node will include string in signed object
+  if (randomBytesToSign) {
+    response.randomBytesToSign = randomBytesToSign
+  }
+
+  if (libs) {
+    response.selectedDiscoveryProvider = libs.discoveryProvider.discoveryProviderEndpoint
+  } else {
+    logger.warn('Health check with no libs')
+  }
+
+  // we have a /db_check route for more granular detail, but the service health check should
+  // also check that the db connection is good. having this in the health_check
+  // allows us to get auto restarts from liveness probes etc if the db connection is down
+  await sequelize.query('SELECT 1')
+
+  if (
+    !response['numberOfCPUs'] || response['numberOfCPUs'] < MIN_NUBMER_OF_CPUS ||
+    !response['totalMemory'] || response['totalMemory'] < MIN_TOTAL_MEMORY ||
+    !response['storagePathSize'] || response['storagePathSize'] < MIN_FILESYSTEM_SIZE
+  ) {
+    response['meetsMinRequirements'] = false
+  } else {
+    response['meetsMinRequirements'] = true
+  }
+
+  // If optional `randomBytesToSign` query param provided, node will include string in signed object
+  if (randomBytesToSign) {
+    response.randomBytesToSign = randomBytesToSign
+  }
+
+  if (libs) {
+    response.selectedDiscoveryProvider = libs.discoveryProvider.discoveryProviderEndpoint
+  } else {
+    logger.warn('Health check with no libs')
+  }
+
+  // we have a /db_check route for more granular detail, but the service health check should
+  // also check that the db connection is good. having this in the health_check
+  // allows us to get auto restarts from liveness probes etc if the db connection is down
+  await sequelize.query('SELECT 1')
+
+  if (
+    !response['numberOfCPUs'] || response['numberOfCPUs'] < MIN_NUBMER_OF_CPUS ||
+    !response['totalMemory'] || response['totalMemory'] < MIN_TOTAL_MEMORY ||
+    !response['storagePathSize'] || response['storagePathSize'] < MIN_FILESYSTEM_SIZE
+  ) {
+    response['meetsMinRequirements'] = false
+  } else {
+    response['meetsMinRequirements'] = true
   }
 
   return response
 }
 
+// TODO remove verbose health check after fully deprecated
+const healthCheckVerbose = async ({ libs, snapbackSM } = {}, logger, sequelize, getMonitors, numberOfCPUs, getTranscodeQueueJobs, getFileProcessingQueueJobs, getAggregateSyncData, getLatestSyncData) => {
```

Comment:
i dont think the final two params are actually consumed in health check. what happens if we enable it?

Reply:
Right, those two params weren't used before either and nothing would happen. I assumed they were left there for some reason or backwards compatibility but I think it should be safe to remove.

Reply:
As opposed to removing, can we add it to the health check and see what happens? 🤷‍♂️

This seems like a case of someone trying to wire it up but forgetting to finish it

Reply:
I know it's not scoped in this pr and not your doing so we could follow up on a separate pr

Reply:
Yup, I'll look into it.

Reply:
It looks like they were added by accident and the data is actually exposed through monitors. I'll confirm and clean it up. https://github.com/AudiusProject/audius-protocol/pull/1537